docs/pages/admin-guides/access-controls/access-graph/ssh-keys-scan.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Discover insecure Access Patterns with Teleport Policy
+description: Describes how to enable SSH Keys Scanning using Teleport Policy and Access Graph.
+---
+
+Organizations often struggle to maintain a secure environment due to the lack of visibility into access patterns.
+Long-lived SSH keys, for example, can pose a significant security risk as they can be forgotten or mismanaged while
+still providing non-auditable access to critical systems.
+
+With Teleport Policy's Access Graph, you can gain insights on how SSH keys are used within your environment. By scanning
+all SSH authorized keys present on your servers and SSH Private Keys present on company managed laptops, Teleport Policy can
+provide a visual representation of the access patterns and help you enhance the security of your environment.
+
+This functionality gives you insights into the following areas:
+
+- Which servers have SSH Authorized Keys that could be used to bypass Teleport?
+- Which users have SSH Private Keys that grant access to SSH servers?
+- Which laptops have unprotected SSH Private Keys?
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product
+available to Teleport Enterprise edition customers.
+
+After logging in to the Teleport UI, navigate to the Management tab. If enabled, Access Graph options can be found
+under the Permission Management section.
+
+## How it works
+
+Teleport and Teleport Policy's Access Graph synchronize various resources, including SSH authorized keys and private keys.
+
+These resources are then visualized using the graph representation detailed in the
+[Access Graph page](../teleport-policy/getting-started-policy.mdx).
+
+### Importing Node's SSH Authorized Keys
+
+Teleport Cluster Nodes running the Teleport SSH Service continuously scan for SSH Authorized Keys present on the server.
+The public fingerprint of these keys is sent to the Teleport Auth Service, which then pushes them to the Access Graph.
+Together with the key fingerprint, the node also sends the following metadata:
+
+- `public_fingerprint`: The SHA256 fingerprint of the public key.
+- `host_id`: The unique identifier of the node.
+- `username`: The local username whose access is granted by the key.
+- `key_comment`: The comment associated with the key in the `authorized_keys` file.
+
+By default, Teleport scans the `$HOME/.ssh/authorized_keys` and `$HOME/.ssh/authorized_keys2` files for authorized keys.
```

Comment:
In what context is `$HOME` evaluated? The home directory for the user that the agent is running as, or all users?

Reply:
Added a note. It expands to the home dir of all users in the machine either being local or ldap managed